cita-executor/core/src/externalities.rs

```diff
@@ -418,6 +423,16 @@ where
 
         let address = self.origin_info.address;
         let balance = self.balance(&address)?;
+
+        if self.chain_version > 1 {
```

Comment:
It is better  to define some `MACRO` for the `version` number, and then comment the difference between each version. I think they will be more readable!

Reply:
In term of protocol version, we had a explanation docs to tell difference between each versions.

As for `Use Macro`, we can discuss details later. Sadly, i am not skilled to macro usage in rust.

https://docs.citahub.com/zh-CN/cita/protocol-upgrade/v0
https://docs.citahub.com/zh-CN/cita/protocol-upgrade/v1

Reply:
#565

Reply:
> In term of protocol version, we had a explanation docs to tell difference between each versions.

The doc and the comment two things.
Will do it at a new pr.